### ISSUE TITLE
Make Brains tests pass

### ISSUE THREAD

**Issue by gaktive** [cloudfoundry-incubator/kubecf#29]
Blocked until #382 is done

**Comment by mook-as** [comment]
Note that this would only make sense once we have the things we test in there.  For example, kubecf currently does not have cf-usb.

**Comment by andreas-kupries** [comment]
Ref #582

**Comment by viovanov** [comment]
List of stuff that's not currently passing:

- [ ] 011_nfspersi_test.rb (in progress)
- [ ] 014_minibroker_mariadb_test.rb (will be tested manually)
- [ ] 015_minibroker_postgres_test.rb (will be tested manually)
- [x] 017_syslog_forwarding_test.rb  (deprecated)
- [ ] 012_minibroker_redis_test.rb (will be tested manually)
- [ ] 016_minibroker_mongodb_test.rb (will be tested manually)

**Comment by f0rmiga** [comment]
@viovanov As per QA request, I enabled the Minibroker tests and exposed some knobs as BOSH properties here: https://github.com/SUSE/brain-tests-release/pull/15.

**Comment by gaktive** [comment]
NFS test no longer required. Please remove it from Brains.

**Comment by fargozhu** [comment]
unblocked with the #382 closed.

**Comment by viovanov** [comment]
@f0rmiga is it safe to assume that this is no longer needed because of MITS?
or do we still want these brain tests for integration?

**Comment by viovanov** [comment]
@satadruroy @gaktive @fargozhu should this be in Jira instead?

**Comment by f0rmiga** [comment]
@viovanov Brains won't have Minibroker tests anymore.

**Comment by fargozhu** [comment]
@viovanov @satadruroy I am closing this one since it is being tracked internally.
